### DISCUSSION TITLE
Creating codespace for repositories not part of the organization?

### DISCUSSION THREAD

**Discussion by julian-hoch** [community/community#37612] [Codespaces] [answered]
Hi, 
I enabled codespace for my organization and added a spending limit, and allowed team members to create codespaces.
I can now create a codespace for repositories that belong to the organization. But for other private repositories which are not part of the organization, but belong to members of the organization directly, I cannot create codespaces.
Is this by design? Does this mean, team members of an organization can ONLY create codespaces for repositories listed as belonging to the organization?
Thanks, Julian

**Comment by ossanna16** [comment]
Hi there @julian-hoch and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by aditya30394** [comment, marked as answer]
Hi,
This behaviour is by design. Enabling Codespaces for an organization only enables Codespace creation on repos in that org.

**Reply by julian-hoch** [reply]
Ok, thanks for letting me know! I guess then I will go with gitpod.io.
Julian
